api/lib/rest-to-tx-converter.js

```diff
@@ -19,12 +19,38 @@ function RestToTxConverter() {}
  *  @param {ripple-lib Transaction} transaction
  */
 RestToTxConverter.prototype.convert = function(payment, callback) {
+  function sendMaxRequired() {
+    var src = payment.source_account;
+
+    var srcamt = payment.source_amount;
```

Comment:
Why'd you remove the camel-casing? Also, I generally prefer fewer abbreviations, though sometimes it's reasonable to abbreviate.